hoomd/mpcd/VirtualParticleFiller.h

```diff
@@ -91,15 +95,11 @@ class PYBIND11_EXPORT VirtualParticleFiller
         unsigned int m_type;            //!< Fill type
         std::shared_ptr<::Variant> m_T; //!< Temperature for filled particles
         unsigned int m_seed;            //!< Seed for PRNG
+        unsigned int m_first_tag;       //!< First tag of locally held particles
```

Comment:
I think this should only be defined for the manual filler, right?

Reply:
Yup. you are correct